src/startupscriptgenerator/common/fsvalidation.go

```diff
@@ -19,6 +19,10 @@ func PathExists(path string) bool {
 
 func FileExists(path string) bool {
 	fi, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		return false
+	}
+
 	if err != nil {
 		return false
 	}
```

Comment:
doesn't it already cover the `IsNotExist` case?

Reply:
Seems like not..at least this is what I read is a recommended pattern.